bin/generatePrivatePage.py

```diff
@@ -94,7 +89,7 @@ def main():
         os_dir = os.path.join(huawei_dir, huawei_os)
         specific_os_dir = [name for name in os.listdir(os_dir) if os.path.isdir(os.path.join(os_dir, name))]
         for directory_specific_os in specific_os_dir:
-            context['huawei'].append({'alphaNumeric': re.sub(r'\W+', '', '{}{}'.format(huawei_os, directory_specific_os)),
+            context['huawei'].append({'alphaNumeric': re.sub(r'\W+', '', os.path.join(huawei_os, directory_specific_os)),
```

Comment:
I think that we are not composing path here - only string

Reply:
Oh yeah, you're right